ThreeBotPackages/threebot/farmmanagement/html/service.js

```diff
@@ -0,0 +1,46 @@
+export default ({
+  getName () {
+    return axios.post(`${window.config.jsApiUrl}/identity/name`, {
```

Comment:
Please update to use gedis client instead of axios

Reply:
Where do I find documentation for the gedis in javascript?

Reply:
I updated it :) https://github.com/threefoldtech/jumpscaleX_threebot#gedis-http-javascript-client